float64[8] gama
---
bool set
